src/main/java/com/moabam/api/domain/member/Member.java

```diff
@@ -121,7 +121,17 @@ public void delete(LocalDateTime now) {
 	}
 
 	public void changeNickName(String nickname) {
-		this.nickname = nickname;
+		this.nickname = requireNonNullElse(nickname, this.nickname);
+	}
+
+	public void changeIntro(String intro) {
+		if (intro != null) {
+			this.intro = intro;
+		}
```

Comment:
```suggestion
                this.intro = requireNonNullElse(intro, this.intro);
```
C: 얘도 동일하게 requireNonNullElse 쓰면 되지 않나여?!

Reply:
그러게요,,, ㅋㅋㅋㅋㅋㅋ 바로 고치겠습니다!